<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>5</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="gameTitle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="browserSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>4</number>
      </property>
      <widget class="GameNotationWidget" name="chessBrowser" native="true"/>
      <widget class="QWidget" name="boxWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="boxPageMove">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>354</width>
             <height>426</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Move</string>
           </attribute>
           <layout class="QGridLayout" name="layoutPageMove"/>
          </widget>
          <widget class="QWidget" name="boxPageEval">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>354</width>
             <height>426</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Evaluation</string>
           </attribute>
           <layout class="QGridLayout" name="layoutPageEval"/>
          </widget>
          <widget class="QWidget" name="boxPageOther">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>354</width>
             <height>426</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Other</string>
           </attribute>
           <layout class="QGridLayout" name="layoutPageOther"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameNotationWidget</class>
   <extends>QWidget</extends>
   <header>gamenotationwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
